graphql-types/package.json

```diff
@@ -15,5 +15,10 @@
   },
   "main": "dist/commercetools-test-data-graphql-types.cjs.js",
   "module": "dist/commercetools-test-data-graphql-types.esm.js",
-  "files": ["dist", "package.json", "LICENSE", "README.md"]
+  "files": ["dist", "package.json", "LICENSE", "README.md"],
+  "dependencies": {
+    "@babel/runtime-corejs3": "^7.21.0",
+    "@graphql-codegen/plugin-helpers": "5.1.0",
+    "lodash": "4.17.21"
+  }
```

Comment:
Are these dependencies still needed?

Reply:
No they are not.

Removed them in this commit: 2b5f91166820a4c7a6282aedf7455403b9a73937